### PR TITLE
Add pro/torch/mxnet extras, update README.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +21,42 @@ GluonTS is a Python package for probabilistic time series modeling, focusing on
...
 
-GluonTS requires Python 3.7, and the easiest way to install it is via `pip`:
+GluonTS requires Python 3.6, and the easiest way to install it is via `pip`:
 
 ```bash
-pip install --upgrade gluonts mxnet~=1.8   # to be able to use MXNet-based models
-pip install --upgrade gluonts torch~=1.10  # to be able to use PyTorch-based models
+# minimal installation
+pip install gluonts
 ```
```
Comment (lostella) [inline README.md:29]: I'm wondering if this should be the first thing one sees two lines below reading "installation", the warning sign in the comment line may go unnoticed

Comment (jaheba) [reply README.md:29]: I can make the warning BOLD!

**Review by lostella — APPROVED** [review]

**Comment by lostella** [comment]
Should these changes also be reflected in the documentation?

**Comment by lostella** [comment]
https://github.com/awslabs/gluon-ts/blob/dev/docs/install.rst

**Pull request merged**